lib/rules/landmark-unique-matches.js

```diff
@@ -42,7 +25,3 @@ function isLandmarkVirtual(vNode) {
 
   return landmarkRoles.indexOf(role) >= 0 || role === 'region';
 }
-
-function isHeaderFooterLandmark(headerFooterElement) {
```

Comment:
As mentioned in a previous comment, this function has apparently been redundant ever since the implicit role mapping was introduced/updated.